docs/concepts/custom-resources.md

```diff
@@ -73,6 +73,10 @@ The below diagram shows how the `KongPlugin` resource can be linked to an
 
```

Comment:
High level comment: the paragraphs here are difficult to understand, and passive voice is a significant contributing factor. I suggest that we try to get rid of all passive voice where it's reasonable.

https://blog.robgarrett.com/2005/09/19/passive-voice-in-professional-documents/

and

> The Microsoft Manual of Style is used by professional technical writers.
> The 4th edition (2012) says:
> > In general, use active voice. Active voice emphasizes the person or thing performing the action. It's more direct than passive voice, which can be confusing or sound formal.
> > - Use passive voice sparingly. It is all right to use passive voice in the following situations:
> > - To avoid a wordy or awkward construction.
> > - When the action, not the doer, is the focus of the sentence.
> > - When the subject is unknown.
> > - In error messages, to avoid giving the impression that the user is to blame.

Reply:
As agreed elsewhere, let's not use blanket rules with this level of generality. Specific suggestions follow.